.gitignore

```diff
@@ -1 +1,4 @@
-node_modules/
\ No newline at end of file
+.firebase/
+dist/
```

Comment:
Hattest du in der Datei keinen Merge Confilct? Wir hatten doch die dist/ Zeile wieder entfernt.

Reply:
Hab jetzt selber die Merge Conflicts aufgelöst.